Source/Core/InputCommon/ControllerInterface/ControllerInterface.h

```diff
@@ -62,11 +62,18 @@ class ControllerInterface : public ciface::Core::DeviceContainer
 
   ControllerInterface() : m_is_init(false) {}
   void Initialize(const WindowSystemInfo& wsi);
-  void ChangeWindow(void* hwnd);
-  void RefreshDevices();
+  // Not thread safe
+  void ChangeWindow(void* hwnd, bool is_exit = false);
```

Comment:
There is only one call site and likely won't be anymore, plus it's false by default so it's not even specified in other places.
Sure it would be better, but I don't think its worth bothering honestly.